src/features/game/types/milestones.ts

```diff
@@ -133,6 +134,24 @@ export const FISH_MILESTONES: Record<MilestoneName, Milestone> = {
       "Luminous Anglerfish Topper": 1,
     },
   },
+  "Deep Sea Diver": {
+    task: "Catch 5 of every fish",
+    percentageComplete: (farmActivity: GameState["farmActivity"]) => {
+      const encyclopediaFish = getEncyclopediaFish();
+      const totalFishRequired = encyclopediaFish.length * 5;
+
+      const totalFishCaught = encyclopediaFish.reduce(
+        (total, name) =>
+          total + Math.min(farmActivity[`${name} Caught`] ?? 0, 1),
```

Comment:
The logic here is incorrect. We will always be adding `1` to the total if we take the `min` value. 
```suggestion
          total + (farmActivity[`${name} Caught`] ?? 0)),
```

Reply:
@spencerdezartsmith That proposed change will mean that you just need to catch `encyclopediaFish.length * 5` anchovies to complete the milestone.

I think the logic he was looking for is this...
```
total + Math.min(farmActivity[`${name} Caught`] ?? 0, 5),
```
... which limits the contribution for any fish species to 5.